nmma/em/model.py

```diff
@@ -99,12 +99,52 @@
     ],
 }
 
-
-class GenericCombineLightCurveModel(object):
+class Mixin:
```

Comment:
This looks good. But could we call this Lightcurve or some other useful name?

Reply:
Mixin is a standard name for a such a base class. Any suggestions for other name?

Reply:
Ah sorry should have been more clear. I meant like LightcurveMixin or similar.